content/04.baselines.md

```diff
@@ -1,14 +1,16 @@
+More concretely, a researcher may wish to implement and fit a logistic regression model using the same software framework planned for DL first to evaluate the classification performance.
+Establishing such a performance baseline may help researchers with assessing the complexity of the task at hand as well as potential technical challenges surrounding data processing-related steps, which can prove useful for implementing more complex DL architectures later on.
+The utility of this approach is evidenced by the recent development of hybrid models that combine DL and simpler models to improve generalization performance, model interpretability, and confidence estimation [@arxiv:1803.04765; @arxiv:1805.11783].
+Depending on the amount and the nature of the available data, as well as the task to be performed, deep learning may not always be able to outperform conventional methods.
+As an illustration, Rajkomar et al. [@doi:10.1038/s41746-018-0029-1] found that simpler baseline models achieved performance comparable with that of DL in a number of clinical prediction tasks using electronic health records, which may be a surprise to many. 
+Another example is provided by Koutsoukas et al., who benchmarked several traditional machine learning approaches against deep neural networks for modeling bioactivity data and found that, while well tuned deep learning approaches tend to outperform conventional classifiers on a moderately sized dataset, simple methods such as Naive Bayes classification tend to outperform deep learning as the noise in the dataset increases [@doi:10.1186/s13321-017-0226-y].
 
-It is worth noting that conventional machine learning methods (e.g., support vector machines, random forests) are also likely to benefit from parameter tuning.
-It can be tempting to train baseline models with these conventional methods using default parameters, which may provide acceptable but not stellar performance, but then tune the parameters for DL models to optimize performance.
-Hu and Greene [@doi:10.1101/385534] discuss a "Continental Breakfast Included" effect by which unequal hyperparameter tuning skews the evaluation of methods, especially those with performance that varies substantially with modest changes to hyperparameters.
-Those wishing to compare methods should tune the parameters of traditional and DL to optimize performance before making claims about relative performance differences.
-The performance comparison among DL models and many other ML approaches is informative only when the models are similarly tuned.
+It is worth noting that conventional off-the-shelf machine learning algorithms (e.g., support vector machines and random forests) are also likely to benefit from parameter tuning.
+It can be tempting to train baseline models with these conventional methods using default parameters, which may provide acceptable but not stellar performance, but then tune the settings for DL algorithms to further optimize performance.
```

Comment:
I would change this back to model, and settings back to parameters. The two are nearly interchangeable in terms of denotation, but I think it's worth describing "algorithmic settings" as parameters since they are specifically chosen (albeit often by a human) based on the training data.

Reply:
> likely to benefit from parameter tuning.

I would rather say change this to "likely to benefit from hyperparameter tuning."

and 

> It can be tempting to train baseline models with these conventional methods using default parameters, 

"to It can be tempting to train baseline models with these conventional methods using default settings," 

since "tuning the parameters of a model" can really mean both, optimizing the weight parameters or changing the learning algorithm's setting

Reply:
We should probably  use whatever wording we use throughout the rest of the text. @Benjamin-Lee Once the core content is added, are we planning to edit things for consistency?

Reply:
@evancofer, yes. Once we are happy with the content, I am planning on doing some passes with an eye towards consistency of style, grammar, etc.

Reply:
clarified it in the first sentence now

> ... a larger number and variety of tuning parameters or algorithm settings (so-called hyperparameters) ...

I think it's useful to be clear by what we mean with tuning parameters because the main audience may not be familiar with jargon like hyperparameters.